google-cloud-spanner/src/main/java/com/google/cloud/spanner/SessionImpl.java

```diff
@@ -464,17 +464,15 @@ public AsyncTransactionManagerImpl transactionManagerAsync(TransactionOption...
 
   @Override
   public ApiFuture<Empty> asyncClose() {
-    XGoogSpannerRequestId reqId =
-        this.getRequestIdCreator().nextRequestId(1 /* TODO: channelId */, 0);
+    XGoogSpannerRequestId reqId = this.getRequestIdCreator().nextRequestId(this.getChannel(), 0);
```

Comment:
Why is the initial attempt 0 here and in the `close()` method, but 1 in `beginTransactionAsync()`?